### DISCUSSION TITLE
課題１４－問題２について（9/7）

### DISCUSSION THREAD

**Discussion by TakedaAyano** [nsc-kiwamu/traning-takeda#74] [Q&A]
問題2についてなのですが、
実行すると以下のエラーが出ます。これを見てみると、
「課題14でインポートされたモジュールが課題13の中で見つからない」という内容になっていることがわかりました。

```
Error [ERR_MODULE_NOT_FOUND]: Cannot find module 'C:\worktraning\traning-takeda\dist\traning13\Common13' imported from C:\worktraning\traning-takeda\dist\traning14\Practice14.js
?[90m    at new NodeError (node:internal/errors:372:5)?[39m
?[90m    at finalizeResolution (node:internal/modules/esm/resolve:437:11)?[39m
?[90m    at moduleResolve (node:internal/modules/esm/resolve:1009:10)?[39m
?[90m    at defaultResolve (node:internal/modules/esm/resolve:1218:11)?[39m
?[90m    at ESMLoader.resolve (node:internal/modules/esm/loader:580:30)?[39m
?[90m    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:294:18)?[39m
?[90m    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:80:40)?[39m
?[90m    at link (node:internal/modules/esm/module_job:78:36)?[39m {
  code: ?[32m'ERR_MODULE_NOT_FOUND'?[39m
}
```
課題14の中で、
import { makeRandomList } from "../traning13/Common13";
と、インポートをしているのですが、何が原因で上記のエラーが出ているのかがわかりません。

**Comment by nsc-kiwamu** [comment]
原因は書いてある通りです。
英語なのであまり注意深く見ていないかもしれないですが、よく見ると簡単な文だと思うので見てみてください。
※翻訳サイトでも翻訳でも良いですが、その場合は「C:\worktraning\traning-takeda\dist\traning13\Common13」とかがうまく翻訳されないので
Cannot find module 'C:\worktraning\traning-takeda\dist\traning13\Common13' imported from C:\worktraning\traning-takeda\dist\traning14\Practice14.js
↓
Cannot find module 'X' imported from Y
とかにすればよいかと。

他の課題のインポート文を見てみれば、何が足りていないかわかると思うので見てみてください。
